tsconfig.json

```diff
@@ -27,7 +23,5 @@
     "strict": true,
     "target": "es5"
   },
-  "include": [
-    "src"
-  ]
+  "include": ["**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx", "**/*.eslintrc.js"]
```

Comment:
If we make this change, can we at least exclude `.yarn/sdks` so that we're not linting/formatting 3rd party code?